CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* **BREAKING** Allow product name in layout header component when public ([PR #1736](https://github.com/alphagov/govuk_publishing_components/pull/1736))
 * Add heading option to panel component ([PR #1741](https://github.com/alphagov/govuk_publishing_components/pull/1741)) MINOR
 * Force contents list title to always be Contents or regional equivalent ([PR #1734](https://github.com/alphagov/govuk_publishing_components/pull/1734)) BREAKING
```

Comment:
Do we want to adjust this to be consistent with the `**BREAKING**` formatting used in your change note?

Reply:
Think I follow, have updated.

Reply:
Sorry I wasn't specific enough - I'd go the other way and append with `**BREAKING**`e.g `* **BREAKING** Force contents list title to always be Contents or regional equivalent ([PR #1734](https://github.com/alphagov/govuk_publishing_components/pull/1734))` since this is consistent with existing breaking changes: eg: [1](https://github.com/alphagov/govuk_publishing_components/blame/3d5d4013fde6c858b9e4e1566a34e256305b046a/CHANGELOG.md#L647) [2](https://github.com/alphagov/govuk_publishing_components/blame/3d5d4013fde6c858b9e4e1566a34e256305b046a/CHANGELOG.md#L695)

Reply:
Ah yes, of course. Done, thanks.